src/woo_publications/fixtures/default_admin_index.json

```diff
@@ -165,6 +165,10 @@
                 "publications",
                 "publication"
             ],
+            [
+                "publications",
+                "publicationidentifier"
+            ],
```

Comment:
do we want to expose this as a separate record in the admin? I think I'd rather go for just an inline in the admin.